torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -44,6 +44,12 @@ constexpr const char* NCCL_DESYNC_DEBUG = "NCCL_DESYNC_DEBUG";
 
 constexpr const char* NCCL_ENABLE_TIMING = "NCCL_ENABLE_TIMING";
 
+constexpr const char* TORCH_NCCL_ENABLE_MONITORING =
+    "TORCH_NCCL_ENABLE_MONITORING";
+
+constexpr const char* TORCH_NCCL_HEARTBEAT_TIMEOUT_S =
+    "TORCH_NCCL_HEARTBEAT_TIMEOUT_S";
```

Comment:
nit: S is unnecessary.

Reply:
lol, i asked him to add it. :( 

my concern is that other timeout in PGNCCL is in ms.  maybe this is too ugly but up to you guys

Reply:
Maybe `TORCH_NCCL_HEARTBEAT_TIMEOUT_SEC`? Well from the perspective of users, we want the environment variable's meaning is intuitive. So it does not cause confusion when they set it and ask questions like "What is the unit of timeout". So I prefer to add extra info here.